.github/BOTMETA.yml

```diff
@@ -115,6 +114,7 @@ files:
     labels: conjur_variable
   $lookups/cyberarkpassword.py:
     notify: cyberark-bizdev
+    maintainers: $team_cyberark_conjur
```

Comment:
I'm not sure whether this makes sense, since these seem to be distinct products.

Reply:
what do you mean? `team_cyberark_conjur` contains two other folks and is used in two places

Reply:
[Conjur](https://www.conjur.org/) (which is open-source) is another product from Cyberark than [Password](https://www.cyberark.com/products/privileged-account-security-solution/enterprise-password-vault/) (which is closed-source). I don't see why the conjur maintainers should have anything to do with the cyberarkpassword lookup, since it is for a different product.

Reply:
removed